scripts/copy_qubit.py

```diff
@@ -72,9 +72,16 @@ def get_data(file_path):
                 read=True
             elif(read and row[sample_index]):
                 #this is every other row
-                data[row[sample_index]]={}
-                data[row[sample_index]]['concentration']=row[conc_index]
-                data[row[sample_index]]['unit']=row[unit_index]
+                if row[sample_index] in data:
+                    #Sample is duplicated, drop the key
+                    log.append("sample {0} has two rows in the Qubit CSV file. Please check the file manually.".format(row[sample_index]))
+                    del data[row[sample_index]]
```

Comment:
Yup, makes sense. Than I guess if a sample occur 3 times it really doesn't matter if it end up in `data`. But if you dont want it to happen one way of many way could be to add another check `list` and chaneging `else` to `elif`. Something like

```
data = {}
duplicated_samples = []
.......
.......
    elif(read and row[sample_index]):
        if row[sample_index] in data:
+                    #Sample is duplicated, drop the key
+                    log.append("sample {0} has two rows in the Qubit CSV file. Please check the file manually.".format(row[sample_index]))
                     duplicated_samples.append(row[sample_index])
+                    del data[row[sample_index]]
        elif row[sample_index] not in duplicated_samples:
.........
.........
    return data
```